wa/instruments/perf.py

```diff
@@ -13,126 +13,385 @@
 # limitations under the License.
 #
 
-
-# pylint: disable=unused-argument
+import itertools
 import os
 import re
+import shlex
 
-from devlib.trace.perf import PerfCollector
-
+from devlib.trace.perf import PerfCollector, PerfCommandDict
 from wa import Instrument, Parameter
 from wa.utils.types import list_or_string, list_of_strs
 
-PERF_COUNT_REGEX = re.compile(r'^(CPU\d+)?\s*(\d+)\s*(.*?)\s*(\[\s*\d+\.\d+%\s*\])?\s*$')
+__all__ = [
+    'PerfInstrument',
+]
+
+DEFAULT_EVENTS = ['migration', 'cs']
```

Comment:
The `migration` default event should be `migrations`.

Reply:
Done